test/scripts/get-missing-cherry-picks.test.ts

```diff
@@ -0,0 +1,235 @@
+import assert from 'assert';
+import {Octokit} from '@octokit/rest';
+import {GetResponseTypeFromEndpointMethod} from '@octokit/types';
+import {spy, when, deepEqual, resetCalls, reset} from 'ts-mockito';
+import {getMissingCommits} from '../../scripts/get-missing-cherry-picks-utils';
+
+describe('get missing cherry-picks tests', () => {
+  const octokit = new Octokit();
+  const spiedRepos = spy(octokit.rest.repos);
+  type CompareResponse = GetResponseTypeFromEndpointMethod<
+    typeof octokit.rest.repos.compareCommitsWithBasehead
+  >;
```

Comment:
How about this? Tested it locally and it works, while ensuring that the Octokit instance is never a real object

```suggestion
import {Octokit, RestEndpointMethodTypes} from '@octokit/rest';
import {spy, when, deepEqual, resetCalls, reset} from 'ts-mockito';
import {getMissingCommits} from '../../scripts/get-missing-cherry-picks-utils';

type CompareResponse =
  RestEndpointMethodTypes['repos']['compareCommitsWithBasehead']['response'];

describe('get missing cherry-picks tests', () => {
  const octokit = {
    rest: {
      repos: {
        compareCommitsWithBasehead: () => null,
      },
    },
  } as unknown as Octokit;
  const spiedRepos = spy(octokit.rest.repos);
```

Reply:
ahmazing, thanks